### PR TITLE
Code Shufflin'

### PR DESCRIPTION
**Pull request by ben-clayton**
First stage in merging the `atom` and `gfxapi` packages.
Most of the key-players in `atom` have been moved.

### CODE REVIEW

File: gapis/gfxapi/test/gfxapi_test_import/imported.go [hunk 1]
```diff
@@ -1,49 +0,0 @@
-// Copyright (C) 2017 Google Inc.
```
Comment (dsrbecky) [inline gapis/gfxapi/test/gfxapi_test_import/imported.go]: This file seems pretty empty, so assume it was dropped on purpose during the move?

File: gapis/atom/list.go [hunk 2]
```diff
@@ -1,59 +0,0 @@
...
-package atom
-
-import (
-	"context"
-
-	"github.com/google/gapid/gapis/api"
-)
-
-// List is a list of atoms.
-type List struct {
```
Comment (dsrbecky) [inline gapis/atom/list.go]: Ohhh, that is kind of pity to see it go. I might add it back as part of GC optimization in distant future.
(provided it proves worthwhile, and provided the commands still live in Go memory by then)

**Comment by dsrbecky** [comment]
Is it really for review?  The PR keeps changing under my feet.

File: gapis/atom/list.go (hunk 2 above)
Comment (dsrbecky) [inline gapis/atom/list.go]: Ohhh, that is kind of pity to see it go. I might add it back as part of GC optimization in distant future.
(provided it proves worthwhile, and provided the commands still live in Go memory by then)

Comment (ben-clayton) [inline gapis/atom/list.go]: Why do you want it? It made use awkward, and hid the fact we were making whole-capture copies of the entire command stream everywhere.

File: gapis/gfxapi/test/gfxapi_test_import/imported.go (hunk 1 above)
Comment (ben-clayton) [inline gapis/gfxapi/test/gfxapi_test_import/imported.go]: Will delete in a followup CL.

Comment (ben-clayton) [inline gapis/gfxapi/test/gfxapi_test_import/imported.go]: Sorry, just properly read the comment. Yeah, it was left over crap from the named imports and wasn't referenced.

**Pull request merged**